toolkit/job-analysis/agenda-for-job-analysis-workshop-day-1.md

```diff
@@ -5,89 +5,172 @@ section: hiring-phases
+        <ul>
+          <li>Time required will depend on SME alignment</li>
+        </ul>
       </td>
     </tr>
     <tr>
-      <th scope="row">11:45 am</th>
+      <th scope="row">1 hour 15 min</th>
       <td>
-        Lunch (1 hour)
+        Define Competencies
+        <ul>
+          <li>Break into small groups to create drafts</li>
+            <ul>
+              <li>Leverage example competancies</li>
+              <li>Definition should be 1-3 sentences, not a list of job tasks</li>
+            </ul>
+          <li>Come back together as a group to validate drafts</li>
+        </ul>
       </td>
     </tr>
     <tr>
-      <th scope="row">12:45 pm</th>
+      <th scope="row">5 min</th>
       <td>
-        Define Competencies (1 hour 15 min)
+        Explain Proficiency Levels
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">1 hour 15 min</th>
+      <td>
+        Create proficiency levels for each competency
         <ul>
-          <li>Use examples if needed from DOI and USDS</li>
-          <li>Definition should be 1-3 sentences, not a list of job tasks</li>
+          <li>Break into groups as before and divide competencies between groups</li>
+          <li>Reference examples and have proficiencies build on each other</li>
         </ul>
       </td>
     </tr>
     <tr>
-      <th scope="row">2:00 pm</th>
+      <th scope="row">45 min</th>
       <td>
-        Understanding Proficiency Levels (5 min)
+        Discuss and finalize proficiency levels with group
+        <ul>
+          <li>Time required will depend on SME alignment</li>
+        </ul>
       </td>
     </tr>
     <tr>
-      <th scope="row">2:05 pm</th>
+      <th scope="row">15 minutes</th>
       <td>
-        Create proficiency levels for each competency (1 hour 15 min)
+        Confidentiality agreements
         <ul>
-          <li>Break into 2 groups and divide competencies between groups</li>
+          <li>SMEs sign privacy agreement on testing materialsw</li>
+            <ul>
+              <li>HR should talk about importance of confidentiality test materials</li>
+              <li>Use this time to ensure everyone knows who is saving/storing the test materials</li>
+            </ul>
         </ul>
       </td>
     </tr>
     <tr>
-      <th scope="row">3:20 pm</th>
+      <th scope="row">30 min</th>
+      <td>
+        Review job tasks against competencies
+          <ul>
+            <li>Only needed if you are on day 2 of the workshop at this point and need to refresh SMEs</li>
+          </ul>
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">1 hour 30 minutes</th>
+      <td>
+        Resume review exercise and competency/proficiency refinement
+        <ul>
+          <li>This is the most important exercise so far - the group will check its work to date</li>
+          <li>Hand SMEs sample resumes (ideally, difficult resumes) and have them review against the current drafts</li>
```

Comment:
i don't love the word "difficult" i would define what that means: have a mix of resumes- some shorter 2 page private sector resumes and some federal style longer resumes; include resumes that are strong, weak, and borderline.

Reply:
edited

Reply:
https://github.com/usds/SME-QA-Website/commit/b23d9164b2e7686f81f5a97604fa6582d6862426